website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -162,9 +176,9 @@ INFO    Browser path: C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.82\c
 ```
 
 ## Selenium Grid
```

Comment:
So, grid is weird right now because you can't use the dynamic aspect of setting the browser version in options. Right now, you need to create a stereotype with the supported versions so that the options will match the node. I think we should change this, but that's still under discussion. 😄 Regardless I think it's a caveat we should spell out.

Reply:
I don't know more about what's going on with Selenium Grid than I explained in this doc. Please extend/change it if required.

Reply:
What I know is explained in this issue - https://github.com/SeleniumHQ/selenium/issues/12530
My lack of grid experience means I don't actually know how to make it work properly right now, either, is part of the problem. ☹️